source/_integrations/matter.markdown

```diff
@@ -240,6 +240,21 @@ In some cases, bridging devices into Home Assistant via Matter might not bring y
 
 <lite-youtube videoid="rEugjMk-4II" videoStartAt="4192" videotitle="Bridge a Matter device to Home Assistant"></lite-youtube>
 
+## Sharing a device from Home Assistant with another platform
```

Comment:
```suggestion
## Sharing a device from Home Assistant to another platform
```

Not sure about this but replacing "with" with "to" makes it more explicit what we mean here ?

Reply:
"share to" sounds like progressive grammar to me. But I get your point. Rephrased it to "share across platforms".